dev.html

```diff
@@ -2799,6 +2952,296 @@ <h2>Conditions</h2>
         </pre>
     </section>
 
+    <section>
+      <h2>RDF-Star</h2>
+
+      An YARRRML-star mapping defines a mapping from any data in a structured source format to RDF-star.
+      It enables the generation of quoted triples in the position of subject and/or object.
+
+      Quoted triples can be created by referencing the mapping identifier using the <code>quoted</code> keyword.
+      It is also possible to create non-asserted quoted triples using the <code>quotedNonAsserted</code> keyword.
+
+      <section>
+        <h2>Keys</h2>
+
+        <dl>
+          <dt><dfn>quoted</dfn></dt>
+          <dd>Quoted mapping.<br />
+            Required profiles: RMLStar. <br />
+            Datatype: Mapping reference.
+          </dd>
+          <dt><dfn>quotedNonAsserted</dfn></dt>
+          <dd>Non-asserted quoted mapping.<br />
+            Required profiles: RMLStar. <br />
+            Datatype: Mapping reference.
+          </dd>
+        </dl>
+      </section>
+
+    
+      <section>
+        <h2>Examples</h2>
+
+
+      <div id="tabs-60" class="nanotabs">
+        <ul>
+          <li><a href="#tab60-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab60-2">RML</a></li>
+        </ul>
+        <div id="tab60-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Quoted mapping in object">
+            mappings:
+              person:
+                subject: http://example.org/person/$(Name)
+                predicateobjects:
+                 - predicates: ex:confirms
+                   objects:
+                    - quoted: student
```

Comment:
How does this differ from a join in YARRRML? The latter are done using functions.
Can we maybe unify the descriptions?

Reply:
My main issue is that new keys `quoted` and `unquoted` are introduced that mean the same as the existing `mapping` one ('the value is the key of a mapping'), but with additional specifiers, so for me it would be more intuitive to write

```yaml
mappings:
  person:
    subject:
      - mapping: project
        interpretAs: quoted # of course, `interpretAs` is a placholder for a better term
```

And if we ever go for the 'result of a join is a literal, not the subject', I would vote for consistency, i.e.,

```yaml
mappings:
  person:
    subject:
      - mapping: project
        interpretAs: join # the default
        butInsteadOfTheSubjectOfThatMappingActuallyUseReference: name
```

It's a bit following Moody's guidelines, but I know it's a matter of personal preference

And to answer your questions:
1. The ID is `student`, just as the value of `$(colleague)~iri` is `$(colleague)`: we probably need to specify that if you want to use `~` you need to escape it (but we can specify that once for the entire YARRRML spec)
2. you need to add `~quoted` only if you quote that mapping. If you don't add anything, the default is what it is now: interpret as a join. I'm not sure I see the potential confusion
3. well, I'm not convinced you need full inline support, see [my comment](https://github.com/kg-construct/yarrrml-spec/pull/4#discussion_r1101558220) ;). Is there a specific reason for it to must be in one line (as my proposed alternative doesn't introduce more characters?)
4. Sure, but that's a matter of validation: keeping the number of potential constructs as low as needed is complementary to validating that your YARRRML is valid. (`- [data/person.json~jsonpath, $]` is a valid construct, but I can only add it under the `sources` key)

Something else to consider: is it sensible to have both `mapping`, `quoted` and `unquoted` fields for the same predicateObject? Which combinations make sense? I assume no combinations are sensible